app/models/solidus_stripe/payment_intent.rb

```diff
@@ -52,10 +52,7 @@ def create_stripe_intent(stripe_intent_options)
             order.currency,
           ),
           currency: order.currency,
-
-          # The capture method should stay manual in order to
-          # avoid capturing the money before the order is completed.
-          capture_method: 'manual',
+          capture_method: payment_method.auto_capture? ? 'automatic' : 'manual',
```

Comment:
Does it fix #178? If so, do we need to implement the unhappy path (see https://github.com/solidusio/solidus_stripe/issues/178#issuecomment-1479410507). IMHO, if so, at least this one should deserve a separate PR for better reference in the future.

Reply:
Agree with @waiting-for-dev here. I'm fine doing this before, and even partially (updating the issue with the remaining things to do), but I'd like this not to be part of the preparatory PR.

Reply:
This was added because the comment didn't seem relevant anymore with the deferred confirm and so it made sense to connect the value to the auto_capture setting instead of just deleting the comment. 
But open to postponing the change, it's not strictly required for the deferred capture PR.